app/services/organizations/default_pet_task_service.rb

```diff
@@ -7,12 +7,15 @@ def initialize(pet)
 
   def default_tasks_array
     @default_pet_tasks.map do |task|
+      next_due = (task.recurring.!) ? nil : task.due_in_days
```

Comment:
This makes me want a scope on default pet task that allows us to call task.recurring? Instead. You don’t have to do it in this PR. But would prefer to read it like English when we can :) Go Ruby

Reply:
Yes that looks much better.